### ISSUE TITLE
[1.16.5] Incompatibility with kubejs scripts

### ISSUE THREAD

**Issue by MohistBOT** [Teneted/Tenet#2260]
## Informations 
**The Minecraft Version** is 1.16.5
**The Mohist Version** is 1055
**Attachments : **https://cdn.discordapp.com/attachments/999706244904407170/999789754608517230/unknown.png, https://cdn.discordapp.com/attachments/999706244904407170/999789841585815653/unknown.png
**Reported by kik0lasi with id 290283526681657354**
## Description 
Can't start all the mods 6 modpack with mohist, always giving error unless I remove the scripts folder. In console it appears as unable to fit X into 3 and in log as error on packet_handler (see images).
## How to reproduce 
Install all the mods and try to start a server

**Comment by MohistBOT** [comment]
**Hello, thanks for opening this issue!**

Before continuing, please understand that Mohist is a **free** and **open source** software, made by people in **their free time**. Know that we are not able to give any eta about when your issue will get answered or fixed.

> Our team will first check if your issue isn't duplicated. If it's the case, we will close your issue, giving your the number of the other issue so that you can follow its progress. If you are able to do it, please check that your issue isn't duplicated, this would help us a lot.

*Thank you for your patience, and have a nice day (or night)!*

**Comment by MohistBOT** [comment]
well, with start server I mean start it and try to join, sorry for not specifying more

**Comment by blueinredout** [comment]
Adding a won't fix tag and closing it with *zero* context seems very unprofessional to me. I would like to at least know what the core issue is, and why anyone even remotely interested in developing Mohist has decided this won't be fixed.
